docs/getting-started.md

```diff
@@ -155,13 +270,13 @@ kubectl create -f https://github.com/kubevirt/kubevirt/releases/download/${KUBEV
 
 # before moving to next step, please make sure that kubevirt pods are running:
 kubectl get pods -n kubevirt -A
-kubectl apply -f https://raw.githubusercontent.com/kubevirt/ovs-cni/master/examples/ovs-cni.yml
```

Comment:
pls remove the opposite (delete) command from the `Purge KubeVirt and Kubernetes` section.